frameworks/kafka/src/main/dist/svc.yml

```diff
@@ -1,24 +1,36 @@
-name: "kafka"
+name: {{FRAMEWORK_NAME}}
+scheduler:
+  principal: {{FRAMEWORK_PRINCIPLE}}
+  zookeeper: {{MESOS_ZOOKEEPER_URI}}
+  api-port: {{PORT_API}}
 pods:
-  kafka:
+  kafka-pod:
```

Comment:
Consider `kafka` instead.  Tasks are named `kafka-pod-0-broker`.  `kafka-0-broker` is probably better.